.github/workflows/linux-tests.yml

```diff
@@ -23,6 +23,8 @@ jobs:
       matrix:
         PYTHON_VERSION: ['3.7', '3.10']
```

Comment:
Could you open a new issue to remove support for Python 3.7 and 3.8? We don't need to support them anymore because they reached end-on-life but we can't make that change in bugfix release (i.e. 1.4.1).